packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -156,15 +157,19 @@ export const useChatMyMessageStyles = makeStyles({
       visibility: 'visible'
     },
     '& msft-mention': {
-      color: '#D83B01',
+      color: tokens.colorStatusWarningBackground3,
       fontWeight: 600
     }
   },
+  attached: {
```

Comment:
It can be renamed to bodyAttached or something like this to show that it's style for the body